Microsoft.HealthVault.Fhir/Constants/HealthVaultVocabularies.cs

```diff
@@ -52,7 +55,12 @@ public static class HealthVaultVocabularies
         public const string SleepJournalAMMedication = "sleep-journal-medication";
 
         public const string IrregularHeartBeatExtensionName = BaseUri + "vital-signs/blood-pressure/irregular-heartbeat";
-     
+
+        public const string ConditionOccurrenceExtensionName = BaseUri + ConditionOccurrence;
```

Comment:
Since these are extensions, can you move them to Microsoft.HealthVault.Fhir/Constants/HealthVaultExtensions.cs and use the URL specified there?

Reply:
We've also been adding the structure definition files in the folder: `Microsoft.HealthVault.Fhir/FhirExtensions/XmlDefinitions`. You can generate it using the tool Forge from Furore.